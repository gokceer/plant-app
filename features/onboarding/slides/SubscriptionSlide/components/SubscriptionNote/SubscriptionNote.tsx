import React from "react";
import { StyleSheet, Text } from "react-native";

const SubscriptionNote = () => {
  return (
    <Text style={styles.note}>
      After the 3-day free trial period you’ll be charged ₺274.99 per year
      unless you cancel before the trial expires. Yearly Subscription is
      Auto-Renewable
    </Text>
  );
};

export default SubscriptionNote;

const styles = StyleSheet.create({
  note: {
    fontFamily: "RubikLight",
    color: "#FFFFFF85",
    fontSize: 10,
    textAlign: "center",
    marginTop: 8,
  },
});
