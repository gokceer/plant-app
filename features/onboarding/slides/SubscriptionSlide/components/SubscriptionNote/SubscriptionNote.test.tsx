import { render } from "@testing-library/react-native";
import React from "react";
import SubscriptionNote from "./SubscriptionNote";

describe("SubscriptionNote", () => {
  it("renders correctly and matches snapshot", () => {
    const { toJSON } = render(<SubscriptionNote />);
    expect(toJSON()).toMatchSnapshot();
  });
});
