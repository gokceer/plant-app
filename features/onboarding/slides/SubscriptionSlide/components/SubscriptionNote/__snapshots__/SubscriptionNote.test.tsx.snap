// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscriptionNote renders correctly and matches snapshot 1`] = `
<Text
  style={
    {
      "color": "#FFFFFF85",
      "fontFamily": "RubikLight",
      "fontSize": 10,
      "marginTop": 8,
      "textAlign": "center",
    }
  }
>
  After the 3-day free trial period you’ll be charged ₺274.99 per year unless you cancel before the trial expires. Yearly Subscription is Auto-Renewable
</Text>
`;
